guides/devboard_as_flasher.rst

```diff
@@ -0,0 +1,64 @@
+Using an ESP devboard as serial flasher
+=======================================
+
+.. _devboard-as-flasher:
+
+If you have a functioning ESP devboard onto which you can flash firmware via USB port,
+it can be also used to flash/communicate via serial to other devices as well.
```

Comment:
```suggestion
ESP development boards usually have an onboard USB interface, either built into the chip (e.g. ESP32-S3) or via a USB-UART bridge chip. However some ESP based devices not designed for development work don't bother with this, and only expose the UART0 pins (TX and RX) for flashing purposes.

Normally you would use a dedicated USB-UART interface board for this but what if you don't have one? In this "emergency" situation it is possible to use a development board that does have a USB-UART bridge chip to flash another device. This is achieved by holding the ESP chip in reset so that it doesn't interfere with the bridge chip operation.

It does NOT require any firmware to be flashed onto the development board and will not change anything already flashed onto it - it's purely a way to use the serial interface chip.

Note: most ESP32 S and C series boards do *not* have a separate USB-UART chip - they have it built into the ESP - so are not suitable for this application.
```